pyblish_lite/window.py

```diff
@@ -627,6 +627,20 @@ def on_item_toggled(self, index, state=None):
 
         index.model().setData(index, state, model.IsChecked)
 
+        data = {"new_value": state, "old_value": not state}
+        if index.data(model.Type) == "instance":
+            instance = self.data["models"]["instances"].items[index.row()]
```

Comment:
This will break once we get a proxy model in there, as it can both rearrange items, remove and add. We'll need to ask the model for the object such that it can make this decision.